app/controllers/provider_foreman_controller.rb

```diff
@@ -403,7 +423,7 @@ def show_record(_id = nil)
     end
 
     if @record.class.base_model.to_s == "ConfiguredSystem"
-      rec_cls = @record.class.base_model.to_s.underscore
+      rec_cls = "#{model_to_name(@record.class.to_s).downcase.tr(' ', '_')}_configured_system"
```

Comment:
Sorry, I'm super confused here.  If I understand correctly `rec_cls` will be `ansible_tower_configured_system` or `foreman_configured_system`.  Then a few lines below you might end up with `provider_foreman_ansible_tower_configured_system`?